web/app.js

```diff
@@ -2656,45 +2659,56 @@ function webViewerWheel(evt) {
       return;
     }
 
-    // It is important that we query deltaMode before delta{X,Y}, so that
-    // Firefox doesn't switch to DOM_DELTA_PIXEL mode for compat with other
-    // browsers, see https://bugzilla.mozilla.org/show_bug.cgi?id=1392460.
-    const deltaMode = evt.deltaMode;
-    const delta = normalizeWheelEventDirection(evt);
     const previousScale = pdfViewer.currentScale;
+    if (isPinchToZoom && supportsPinchToZoom) {
+      // The following formula is a bit strange but it comes from:
+      // https://searchfox.org/mozilla-central/rev/d62c4c4d5547064487006a1506287da394b64724/widget/InputData.cpp#618-626
+      const scaleFactor = Math.exp(-evt.deltaY / 100);
```

Comment:
Although, isn't there a risk that this can be too small to be meaningful; do we perhaps want a check similar to this one as well?
https://github.com/mozilla/pdf.js/blob/d6f63b5fb7cc63fa5d38ff71a56b8ee46d4f1b33/web/app.js#L2753-L2756